CHANGELOG.md

```diff
@@ -0,0 +1,8 @@
+# Changelog
+
+## Version 1.7.0
+
+*June 21, 2022*
+
+- Bump liquid dependency from `4.0.3` to `5.3.0`
+- Drop tzinfo dependency from `2` to `1.1`
```

Comment:
This was upgraded so we could get to Rails 6.1. Is it a good idea to undo this?
https://github.com/DripEmail/condensation/pull/3

Reply:
Could this be 1.5.1? Since 1.5.0 is on the old tzinfo version we'd avoid a breaking change in rails support. This would also mean we won't conflict with releases from the original repo

Reply:
I don't understand the question. What was at 1.5.0?

Reply:
tzinfo was at version 1.1 https://github.com/DripEmail/condensation/blob/v1.5.0/condensation.gemspec#L26

Reply:
Oh I see. The tags don't seem to have been kept up with and the maintainer of this repo bumped it to 1.6.0 a while ago. https://github.com/DripEmail/condensation/commit/57e3f109f733dd888499c2c3ceb56efee252bdde
To your point, we could do an x.x.1, but I thought those types of version bumps were more for patches. 

The parent repo hasn't been updated in nearly five years, so I'm not concerned about conflicts. Looks like Taylor and Morgan had some open PRs to get some of his changes merged, but closed them. Maybe @morganbird can say more about that.
https://github.com/derrickreimer/condensation/pulls?q=is%3Apr+is%3Aclosed